src/query/test/compatibility/test.go

```diff
@@ -562,7 +562,7 @@ func almostEqual(a, b float64) bool {
 	diff := math.Abs(a - b)
 
 	if a == 0 || b == 0 || diff < minNormal {
-		return diff < epsilon
+		return diff < epsilon*minNormal
```

Comment:
This seems a bit weird, thought that `minNormal` is the lowest possible float, although guess this is what Prom does in their tests too 🤷‍♂️

Reply:
TIL: min-normal is not the lowest one, [a nice description in here](https://programming.guide/normal-vs-subnormal-floats.html). Seems that when you go below that, you start losing precision (number of digits) and there might be [performance penalty](https://en.wikipedia.org/wiki/Subnormal_numbers#Performance_issues).